internal/static/testdata/apps/invalid-syntax-app/app.go

```diff
@@ -0,0 +1,22 @@
+package impl
+
+import "github.com/dogmatiq/dogma"
+
+const (
+	// AppKey is the application key.
+	AppKey = "7c3c67dd-6c0b-4952-97d5-54c75fc7a1c6"
+)
+
+// Below is the deliberate illegal Go syntax to test loading of the packages
+// with errors.
+<This is the deliberate illegal Go syntax>
+
+
+// App implements dogma.Application interface.
+type App struct{}
```

Comment:
I think this test probably belongs outside the `apps` sub-folder. I don't think there's any reason it needs to contain ann `App` type or any reference to Dogma, etc, at all is there - it's just about making sure that we don't choke on invalid syntax?

Reply:
Yeah, that's exactly what this test is for. This test also covers that `if` condition that checks if the package is returned as `nil` from the call to `ssautil.AllPackages()` due to the errors in the package.

Reply:
This has been addressed in https://github.com/dogmatiq/configkit/pull/118/commits/883911119e53f568864c0e444f4ce731b7c71d62. However I was unable to get rid of the Dogma package import to cover the `nil` package check. This is because Dogma import check precedes it.